docs/Guide/Comments.md

```diff
@@ -0,0 +1,32 @@
+### Single Line Comments
+
+Everything that comes after a hash character (#) or a double backslace (//) is ignored until the end of the line.
```

Comment:
First, there's no such thing as a "backslace" (typo for "backslash?")  Second, this is a double *forward* slash.